src/NodeParser/MappedTypeNodeParser.ts

```diff
@@ -24,23 +24,91 @@ export class MappedTypeNodeParser implements SubNodeParser {
         );
     }
 
-    private getObjectProperty(type: any, node: ts.MappedTypeNode, context: Context) {
-        return new ObjectProperty(
-            type.value,
-            this.childNodeParser.createType(node.type!, context),
-            !node.questionToken,
-        );
-    }
+    /*
```

Comment:
Let's remove or move this comment.

Reply:
I found it helpful.. but if you really like I can remove it. Maybe you know a better way to write the same comment?